<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="seedSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="seedLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter game seed:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="exponentSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exponentLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter goal number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="exponentLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(e.g. 11 for a goal of 2ยนยน = 2048)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="exponentLabel_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>63</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="goalLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="boardsideSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="boardsideLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter side of </string>
    </property>
   </widget>
   <widget class="QLabel" name="boardsideLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>gameboard:</string>
    </property>
   </widget>
   <widget class="QLabel" name="boardsideLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>159</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(e.g. 4 for a 4x4 board)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftPushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightPushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QLabel" name="victoryLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>You won! :)</string>
    </property>
   </widget>
   <widget class="QLabel" name="defeatLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>You lost! :p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitPushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
